### ISSUE TITLE
Wrong theme+schema when viewing single post as a non-member

### ISSUE THREAD

**Issue by jeroenpraat** [redmatrix/hubzilla#251]
When viewing a single post (the one you see when you click on 'Link to Source') as a non-member (or when not logged in) the post is displayed with the default system theme+schema. And not (as expected) with the theme+schema of the channel the post belongs to.

Example:

https://hubzilla.nl/display/9ab638582c153cd4bb0ce6a786b252e7f41a966b8b58154dbf9f73b648c95ead@hubzilla.nl

Try to view that one ^ logged in and logged out (better to do this in a private browser window).

**Comment by redmatrix** [comment]
There seems to be a regression here; I noticed it yesterday. It may have occurred some time ago because nothing has touched related code in the immediate past. 

The logic was originally that we used llink (local link) in some cases (link to thread from search results), but displayed the 'plink' (permalink) in other cases (Link to Source and everything else). The permalink should link to the copy on the original channel. The local link goes to /display on your site and shows no profile info. I'll have to revisit this and figure out what changed. I'm not seeing the permalink anywhere any more; either that or we're storing the wrong thing in plink (/display/abc123 instead of /channel/xxx/?mid=abc123 )for some reason. 

Quite busy the next couple of days so will get to this in due course but you're welcome to try and find out which of these things is actually happening.
